Assets/LeapMotion/Scripts/Animation/TransformTweenBehaviour.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Leap.Unity.Animation {
+
+  public class TransformTweenBehaviour : MonoBehaviour {
```

Comment:
K, I fleshed out a lot of things for it so we feel comfortable with it as a real script